src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we have already use the application.properties instead hibernate.cfg.xml. So we can delete this file